python/hail/methods/statgen.py

```diff
@@ -2242,7 +2242,7 @@ def realized_relationship_matrix(call_expr):
 
     return KinshipMatrix._from_block_matrix(tstr,
                                             rrm,
-                                            dataset['s'].collect(),
+                                            dataset.col_key[0].collect(),
```

Comment:
Is this guaranteed to always be the first column key? What if there are multiple column keys?

Reply:
RRM expects one key of type str. I'll make sure the appropriate check is in place.